Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml

```diff
@@ -5459,7 +5491,6 @@ script:
       description: Flag for the rate limit retry.
       name: ran_once_flag
     description: Collect and download an investigation package from a machine.
-    polling: true
```

Comment:
This is polling command, we shouldn't remove it.

Reply:
Wich command is polling? I see two here.